src/components/sections/AboutSection.tsx

```diff
@@ -1,13 +1,27 @@
+import { useEffect, useState } from 'react';
 import { AboutCard } from '../about/AboutCard';
 import { DresscodeCard } from '../about/DresscodeCard';
-
 import { ProgramItemList } from '../about/ProgramItemList';
 import { InfoCard } from '../cards/InfoCard';
 import { Section } from '../shared/Section';
-
 import aboutImg from '../../assets/img/about-socha.png';
 
 export const AboutSection = () => {
+  const [isMobile, setIsMobile] = useState(false);
```

Comment:
no need to use hooks and states for this.

we can just use media queries directly in the className (`hidden desktop:block`)